packages/api/src/repository/Feed.ts

```diff
@@ -62,4 +66,14 @@ export class FeedRepository {
       return { ...feed, id: feed._id.toString() }
     }
   }
+
+  private isValidFeed (feed: Omit<FeedDbObject, '_id'>): boolean {
+    return !!(
+      feed.address &&
+      feed.label &&
+      feed.name &&
+      feed.network &&
+      feed.requests
+    )
```

Comment:
Alternative option just in case we want not to risk inadvertently breaking this functionality if we add more keys in the future and forget to add them here:
```suggestion
    return !!Object.values(feed).reduce(prev, val => prev && val)
```

Full disclosure: this approach is less performant though, but the difference is probably negligible unless applied on an object with a million entries, but Typescipt is already preventing that to happen at compile time because of the type signature of the function.